tools/packaging/cpt.py

```diff
@@ -494,7 +494,7 @@ def compile(arg, build_libcpp):
 
     build = Build()
     cmake_config_flags = (srcdir + ' -DLLVM_BUILD_TOOLS=Off -DCMAKE_BUILD_TYPE={0} -DCMAKE_INSTALL_PREFIX={1} '
-                          .format(build.buildType, TMP_PREFIX) + ' -DLLVM_TARGETS_TO_BUILD=host ' +
+                          .format(build.buildType, TMP_PREFIX) + ' -DLLVM_TARGETS_TO_BUILD="host;NVPTX" ' +
```

Comment:
I'd prefer doing that at the `CMakeLists.txt` level of cling, given that - if not using cpt.py - people must *know* to add `NVPTX`. Can cling add `NVPTX` as `LLVM_TARGETS_TO_BUILD`?

Reply:
Yeah, that's a problem. Unfortunately I cannot solve the problem in the `CMakeLists.txt` of cling, because it has to be activated in the clang build too. I think the best way to activate the `NVPTX` backend automatically is to add some code to llvm's `CMakeLists.txt`. Then it will also be enabled in the Clang and Cling project. Unfortunately, this will be a commit, which will not go upstream.